tools/xsetwacom.c

```diff
@@ -47,6 +47,8 @@
 #define safe_realloc(p, n, s) \
 	((size_t)-1 / (n) < (s) ? NULL : realloc((p), (n)*(s)))
 
+#define EXIT_INVALID_USAGE 2;
```

Comment:
hmm, the compiler should warn about this, or at least about a double semicolon when you do this:
```c
   return EXIT_INVALID_USAGE;
```
because it translates to:
```c
   return 2;;
```

As a general rule: *never* terminate a `#define` with a semicolon, unless there's a really good reason for it.

Reply:
That's my bad, didn't catch that when writing it.